geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/package.scala

```diff
@@ -44,17 +44,14 @@ package object accumulo {
     }
 
     object BatchWriterProperties {
-      // Measured in millis, default 60 seconds
-      val WRITER_LATENCY_MILLIS  = SystemProperty("geomesa.batchwriter.latency.millis", (60 * 1000L).toString)
-      // Measured in bytes, default 50mb (see Accumulo BatchWriterConfig)
-      val WRITER_MEMORY_BYTES    = SystemProperty("geomesa.batchwriter.memory", (50 * 1024 * 1024L).toString)
-      val WRITER_THREADS         = SystemProperty("geomesa.batchwriter.maxthreads", "10")
-      // Timeout measured in seconds.  Likely unnecessary.
-      val WRITE_TIMEOUT_MILLIS   = SystemProperty("geomesa.batchwriter.timeout.millis", null)
+      val WRITER_LATENCY      = SystemProperty("geomesa.batchwriter.latency", "60 seconds")
+      val WRITER_MEMORY_BYTES = SystemProperty("geomesa.batchwriter.memory", "50mb")
```

Comment:
Do the old/previous batch writer options still work?  If not, it might be worth adding that to the migration/update guide.

Reply:
yeah, the old values don't work. there is a section already in the upgrade guide on the changes though.